docs/operation/cluster_and_service_configuration.rst

```diff
@@ -0,0 +1,64 @@
+Cluster and Services Configurations
+===================================
+
+This section contains information describing the list of available Cluster and
+Services Configurations, the :term:`Namespace` to which they belong,
+the :term:`ConfigMap` where these configurations are store and finally the
+different ways of customizing and applying any given Cluster and Services
+Configurations.
+
+Managing Cluster and Services Configurations
+********************************************
+
+Newly deployed **MetalK8s** clusters come with chosen default values for most
+cluster services. These default values are totally transparent to Admin users
+and can be customized at any point in time given that Admins follow the
+documented procedure to the later.
```

Comment:
procedure? do we have it?

Reply:
We will have one, there is a dedicated ticket for this.

Here: https://github.com/scality/metalk8s/issues/2263